### ISSUE TITLE
Configuration Description: Unclear wording for callback and logout url

### ISSUE THREAD

**Issue by AugmentMo** [auth0-samples/auth0-ios-swift-sample#105]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0-samples/auth0-ios-swift-sample/tree/master/Sample-01#readme) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0-samples/auth0-ios-swift-sample/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

In the Sample 1 README Configuration description.

"Step 2.
Go to the settings page of your [Auth0 application](https://manage.auth0.com/#/applications/) and add the corresponding URL to Allowed Callback URLs and Allowed Logout URLs, [...]"

It is not necessary clear that you have to put in the same url for Callback and Logout URLs. The wording "Corresponding" might suggest that they have to be adapted. While this may be the case in general, it doesnt seem to be the case for this small tutorial.

### Describe the ideal solution

I recommend adding a sentence that clarifying that for this sample the callback and logout url are the same with "/callback" suffix. Might result in a smoother initial test run for some people (like me).

**Comment by Widcket** [comment]
Hi @AugmentMo, thanks for raising this.

> The wording "Corresponding" might suggest that they have to be adapted.

It's because you have to pick the correct URL for the platform you want to use (iOS or macOS). I'll see how I can reword it to make this clearer.

<img width="1048" alt="Screenshot 2023-06-20 at 11 24 47 PM" src="https://github.com/auth0-samples/auth0-ios-swift-sample/assets/5055789/db80d7d5-7640-4092-86d0-92e02e76ff25">

**Comment by Widcket** [comment]
This has been addressed in #108.
